src/sentry/integrations/slack/command_endpoint.py

```diff
@@ -1,25 +1,41 @@
 import logging
 from typing import Mapping
 
-from django.http import HttpResponse
+from django.http import Http404, HttpResponse
 from rest_framework.request import Request
 
 from sentry.api.base import Endpoint
+from sentry.integrations.slack.link_team import build_linking_url
 from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
 from sentry.integrations.slack.requests.base import SlackRequestError
 from sentry.integrations.slack.requests.command import SlackCommandRequest
+from sentry.models import Identity, IdentityProvider
 
 logger = logging.getLogger("sentry.integrations.slack")
+LINK_TEAM_MESSAGE = "Link your Sentry team to this Slack channel! <{associate_url}|Link your team now> to receive notifications of issues in Sentry in Slack."
+LINK_USER_MESSAGE = "You must first link your identity to Sentry by typing /sentry link. Be aware that you must be an admin or higher in your Sentry organization to link your team."
 
 
-def get_command(payload: Mapping[str, str]) -> str:
-    return payload.get("text", "").split(" ")[0].lower()
+def get_command_and_args(payload: Mapping[str, str]) -> str:
```

Comment:
I think this would be a better way to get the args:
```python
def get_command_and_args(payload: Mapping[str, str]) -> Tuple[str, Sequence[str]]:
    text = payload.get("text", "").lower().split()
	if not text:
		return "", []

    return text[0], text[1:]
```
- `.lower()` will let the user do `/sentry HELP`. 
- `.split()` with no params splits on whitespace so `/sentry<space><space>help` will still work.

You'd also have to do:
```python
if args and args[0] == "team":
```